docs/mast/mast.rst

```diff
@@ -938,6 +995,14 @@ not explicitly called for TICA.
      1  PIXELS        1 BinTableHDU    280   1196R x 12C   [D, E, J, 25J, 25E, 25E, 25E, 25E, J, E, E, 38A]
      2  APERTURE      1 ImageHDU        81   (5, 5)   int32
 
+The SkyCoord constructor requires the 'unit' parameter. Unspecified units will result in an error.
+
+.. doctest-remote-data::
+
+   >>> cutout_coord = SkyCoord(107.18696, -70.50919)
+   ...
+   UnitTypeError: Longitude instances require units equivalent to 'rad', but no unit was given.
```

Comment:
Not sure if you need this example since this is an error handling case for `SkyCoord` and not `astroquery.mast`. This is slightly out of scope.